src/python/WMQuality/Emulators/SiteDBClient/MakeSiteDBMockFiles.py

```diff
@@ -29,13 +31,21 @@
     siteDB = SiteDBAPI()
     for call in calls:
         signature = str(sorted(call.iteritems()))
+        callname = call['callname']
         try:
             result = siteDB.getJSON(**call)
         except HTTPError:
             result = 'Raises HTTPError'
 
         if call['callname'] == 'people':
             result = [res  for res in result for dn in dns if res['dn'] == dn]
+        elif (callname == 'site-names'):
+            for extra in extras[callname]:
+                result.append(extra)
+        else:
+            result = result
+            
+
         lookup.update({signature:result})
     with open(outFilename, 'w') as mockData:
         json.dump(lookup, mockData, indent=1, separators=(',', ': '))
```

Comment:
There is a way to sort it. You can see that in the newer version of DBS that makes its file.